app/scripts/shell/dimSearchFilter.directive.js

```diff
@@ -332,6 +333,29 @@
 
         return (item.classType === value);
       },
+      glimmer: function(predicate, item) {
+        var boosts = [
+          1043138475, // -black-wax-idol
+          1772853454, // -blue-polyphage
+          3783295803, // -ether-seeds
+          3446457162  // -resupply-codes
```

Comment:
Is there anything on the item themselves that hints at their glimmeryness? Less hardcoding will help for RoI